docs/source/acknowledgements.rst

```diff
@@ -0,0 +1,24 @@
+Acknowledgments
+===============
+
+DataLad development is being performed as part of a US-German collaboration in
+computational neuroscience (CRCNS) project "DataGit: converging catalogues,
+warehouses, and deployment logistics into a federated 'data distribution'"
+(Halchenko_/Hanke_), co-funded by the US National Science Foundation (`NSF
+1429999`_) and the German Federal Ministry of Education and Research (`BMBF
+01GQ1411`_). Additional support is provided by the German federal state of
+Saxony-Anhalt and the European Regional Development
+Fund (ERDF), Project: `Center for Behavioral Brain Sciences`_, Imaging Platform
+
+DataLad is built atop the git-annex_ software that is being developed and
+maintained by `Joey Hess`_.
+
+The extension was created during the OHBM Hackathon 2020 and wouldn't have been possible without a `dedicated team of volunteers <https://github.com/datalad/datalad-osf#contributors->`_.
+
+.. _Halchenko: http://haxbylab.dartmouth.edu/ppl/yarik.html
+.. _Hanke: http://www.psychoinformatics.de
+.. _NSF 1429999: http://www.nsf.gov/awardsearch/showAward?AWD_ID=1429999
+.. _BMBF 01GQ1411: http://www.gesundheitsforschung-bmbf.de/de/2550.php
+.. _Center for Behavioral Brain Sciences: http://cbbs.eu/en/
+.. _git-annex: http://git-annex.branchable.com
+.. _Joey Hess: https://joeyh.name
```

Comment:
We should use the current numbers and links from https://www.datalad.org/about.html

DataLad - a decentralized system for integrated discovery, management, and publication of digital objects of science (Halchenko / Hanke), co-funded by the US National Science Foundation (NSF 1912266) and the German Federal Ministry of Education and Research (BMBF 01GQ1905).

NSF 1912266: http://www.nsf.gov/awardsearch/showAward?AWD_ID=1912266

Reply:
done